mutagen.yml

```diff
@@ -1,14 +1,13 @@
 # Set up the Mutagen service and code volume before creating sessions.
 beforeCreate:
-  - cp bundles/storefront.yml docker-compose.yml
   - bash init_project
-  - docker-compose up --build --detach mutagen
-
-# Set up the main services after creating sessions. At this point, sessions will
+  - docker-compose -f bundles/monolith.yml up --build  --force-recreate --detach mutagen
+# Set up the main services (monolith, db, redis, elastic) after creating sessions. At this point, sessions will
 # have been established and code pushed to the shared volume.
+# Specific service will be up and run during reinstall phase
 afterCreate:
-  - docker-compose up --build --detach
-  - bash reinstall
+  - docker-compose -f bundles/monolith.yml up --build --detach
```

Comment:
maybe you need remove this line, because duplicate of https://github.com/duhon/magento-docker/pull/38/files#diff-76fd6caf27fce61d65d476844d72e0c5f6942a7d0e2b0ba49ce0e0017661963dR18